### ISSUE TITLE
Your perspective map is  not correct!

### ISSUE THREAD

**Issue by Naruto-Sasuke** [miao0913/PACNN-PerspectiveAware-CrowdCounting#3]
For example, `IMG_295` in ShanghaiA training set, the size is `1024*684(W*H)`. Your corresponding perspective map is `1024*684(H*W)`.  Referring the ground truth image, the width is larger than height. You mistake width and height.     
Besides,  if you watch the values `perspective_value[0:10, 0]`and `perspective_value[0:10, 1]`, you will see two lines of values which each element in each line shares the same value.  It is wrong!  As the perspective hold the hypothesis that each line shares of the image has the same depth. According to the hypothesis, it should be `perspective[0, 0:10]` is a set of same values.

From the two points above, your perspective is not correct!

**Comment by miao0913** [comment]
Hi, 
I am not clear of what you mean by "each line shares of the image has the same depth". 
BUT I checked my data, it is CORRECT !

For example, IMG_295, its perspective map is of size 684*1024 (H*W) which is the same size of the image (please open the picture to have a look at its width (x-axis) and height (y-axis) carefully). Its values are linearly increased from the top to the bottom along the y-axis while the values in the same row (x-axis) are equal. This is exactly what stated in the paper Page 3).

**Comment by miao0913** [comment]
Plus, I load the data by MATLAB. Be careful with the loading function you choose in python, column and rows might be swaped.
